### ISSUE TITLE
Error in ar_wordcloud installation

### ISSUE THREAD

**Issue by Nuhagh** [iamaziz/ar_wordcloud#1]
I like the output. Thank you for ur efforts. Unfortunately, I could not try it (I used Google colab):
_ERROR: Could not find a version that satisfies the requirement ar_wordcloud (from versions: none)
ERROR: No matching distribution found for ar_wordcloud_
Maybe bc of the dependencies. I didn't find setup.py nor requirements.txt.

**Comment by iamaziz** [comment]
Hey, thanks for giving it a try. Yea, I noticed the installation issue on Colab! I'm glad someone asked though. The package was published to PyPI just yesterday. Perhaps there's some lag in the indexing Colab <--> PyPI, not sure!? However, I think `pip install ar_wordcloud` does work outside of Colab. 

MyBinder is another alternative for Colab. The installation works there I just tried it, here:
https://mybinder.org/v2/gh/iamaziz/ar_wordcloud/master just start a new notebook from there, and `pip install ar_wordcloud` then all good to go.


The setup and requirements info in this file:
https://github.com/iamaziz/ar_wordcloud/blob/master/pyproject.toml#L11-L14
